hooks/usePoll.ts

```
@@ -0,0 +1,91 @@
+  isActive?: boolean;
+  userVoted?: boolean;
+  userCreated?: boolean;
+  sortBy?: string;
+  sortOrder?: "asc" | "desc";
+}
+
+export const usePoll = () => {
+  const queryClient = useQueryClient();
+
+  const getPolls = (filters: UsePollParams = {}) => {
+    return useQuery({
```

ESLint

[error] React Hook "useQuery" is called in function "getPolls" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)